README.md

```diff
@@ -4,6 +4,11 @@ API for serializing and de-serializing JavaScript objects using the
 addon compared to the native `JSON` object isn't too bad, and the space
 required for serialized data is far less than JSON.
 
+### Extension: support for undefined
+The standard msgpack protocol turns `undefined` into `null`.
+This branch adopts (creationix's extension)[https://github.com/creationix/msgpack-js-browser#extension]
+for `undefined` and uses for it the reserved value '0xc4'
```

Comment:
I'm not sure what that like is. Maybe @hmalphettes will have some hints?
Also, I've updated the code yesterday to use `0xc1` : https://github.com/sutoiku/node-msgpack/commit/b3fccfd1becb2db1f5d57f5af136945a8a6e256f